node/clients/profile/profileV1.ts

```diff
@@ -36,9 +36,13 @@ export class ProfileClient extends JanusClient {
           extraFields: customFields,
         },
       }
-    )
+    ).then((profile) => {
+      profile.pii = false
+
+      return profile
+    })
 
-  public getUserAddresses = (user: CurrentProfile) =>
+  public getUserAddresses = (user: CurrentProfile, _: Profile) =>
```

Comment:
Does this extra parameter is necessary?

Reply:
It does because ProfileV2 receives it. Unfortunately, I had to add here and ignore it.